spec/support/system_helpers.rb

```diff
@@ -148,16 +148,11 @@ def resize_window(width: nil, height: nil)
   end
 
   def using_browser_timezone(timezone, &example)
-    previous_browser_timezone = ENV["TZ"]
```

Comment:
For whatever reason, specifying http_client in the driver option makes TZ env var not work 🤷 anyway, while this requires a new dependency (selenium-devtools) it's kinda cleaner